### PR TITLE
Use a new constants file for quick acceptance tests

### LINKED ISSUE #2556: Acceptance Tests for Finalisation
### Description
Add automated tests that create a small network and waits for it to finalise.  Since this is slow with minimal spec we need to either run it as a nightly test or preferably define a micro network with very short block and epoch lengths so finalisation happens faster.

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Use a new constants file for quick acceptance tests.

## Fixed Issue(s)

Fixes #2556

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: util/src/main/resources/tech/pegasys/teku/util/config/swift.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# Minimal preset
```
Comment (mbaxter) [inline util/src/main/resources/tech/pegasys/teku/util/config/swift.yaml]: Might be helpful to add some comments that mark which fields differ from minimal, and some general comments at the top about what this config is for and the fact that it is based on minimal

**Review by mbaxter — APPROVED** [review]
LGTM - maybe we could reenable the [sync test](https://github.com/PegaSysEng/teku/blob/b9e2fbe6ae6a7a9d544d7e4737d05accdfcd5389/acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/SyncAcceptanceTest.java#L28) now as well?

**Comment by cemozerr** [comment]
@mbaxter I tried re-enabling the sync test today, yet its failing. Not sure if it's an issue with the changes I've made in this PR or not. Will investigate more.

File: util/src/main/resources/tech/pegasys/teku/util/config/swift.yaml (hunk 1 above)
Comment (cemozerr) [reply util/src/main/resources/tech/pegasys/teku/util/config/swift.yaml]: Makes sense.

Comment (cemozerr) [reply util/src/main/resources/tech/pegasys/teku/util/config/swift.yaml]: Done.

**Comment by cemozerr** [comment]
Fixed the issue with the sync related test. Seemed like it was failing due to legacy code.

**Comment by mbaxter** [comment]
> Fixed the issue with the sync related test.

Nice!! 🎉

**Pull request merged**